interface-definitions/dhcp-server.xml.in

```diff
@@ -316,6 +316,19 @@
                           </constraint>
                         </properties>
                       </leafNode>
+                      <leafNode name="identifier">
```

Comment:
We already have this:


```
    <leafNode name="duid">
      <properties>
        <help>DHCP unique identifier (DUID) to be sent by dhcpv6 client</help>
        <valueHelp>
          <format>duid</format>
          <description>DHCP unique identifier (DUID)</description>
        </valueHelp>
        <constraint>
          <validator name="ipv6-duid"/>
        </constraint>
      </properties>
    </leafNode>
```

in `interface/dhcpv6-options.xml.i` - maybe rename this node here and make it a common XML include block?

Reply:
I have a follow-up PR lined up for refactoring the XML blocks that include this one.
One reason I held back is because:
1. the regex pattern for `ipv6-duid` validator is slightly different. It won't match `aa:bb:c:d`, but current `identifier` node honors that. I need to have migration take of that.
2. `ipv6-duid` is a full python script. I think we can convert that into simple shell script having `${vyos_libexec_dir}/validate-value --regex ...`.

I plan to tackle them together (including migration).

Reply:
I'm adding migration from `identifier` to `duid` in this PR instead of a followup PR. Converting this to draft till migration scripts are ready.